Bug: 579499 - determine if a filter has been set on the MarkerSupport…

eobrienPilz opened the pull request:
…View

jukzi commented:
@eobrienPilz  
Please add also 2 tests, that assertTrue on the conditions.
And change the commit message to something that shows, that an API is added. like
"Bug 579499 - extend ExtendedMarkersView API"
BTW bug link: https://bugs.eclipse.org/bugs/show_bug.cgi?id=579499

jukzi commented on tests/org.eclipse.ui.tests/Eclipse UI Tests/org/eclipse/ui/tests/markers/MarkerSupportViewTests.java:
```diff
@@ -0,0 +1,247 @@
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors: IBM Corporation - initial API and implementation
```
```suggestion
 * Contributors: Enda O'Brien - initial API and implementation
```

jukzi commented on tests/org.eclipse.ui.tests/Eclipse UI Tests/org/eclipse/ui/tests/markers/MarkerSupportViewTests.java:
```diff
@@ -0,0 +1,247 @@
+
+/**
+ *
+ * @since 3.5
```
remove that  since 3.5

jukzi commented on tests/org.eclipse.ui.tests/Eclipse UI Tests/org/eclipse/ui/tests/markers/MarkerSupportViewTests.java:
```diff
@@ -0,0 +1,247 @@
+public class MarkerSupportViewTests extends UITestCase {
+
+	/**
+	 * @param testName
```
remove that comment.

jukzi reviewed (changes requested):
Please leave the filename as singular "MarkerSupportViewTest" and squash the changes to a single commit (force overwrite).

eobrienPilz committed:
Bug: 579499 - make available 2 methods on ExtendedMarkersView API

jukzi commented:
@eobrienPilz please also prepare a PR for https://github.com/eclipse-platform/www.eclipse.org-eclipse-news/blob/master/4.26/platform_isv.html
to describe the new API

eobrienPilz commented:
> @eobrienPilz please also prepare a PR for https://github.com/eclipse-platform/www.eclipse.org-eclipse-news/blob/master/4.26/platform_isv.html to describe the new API

What's the process ? Will I update that file with the API description and include the change as part of this review ?

jukzi commented:
> What's the process ? Will I update that file with the API description and include the change as part of this review ?

We will finish this review. Afterwards we commit the N&N entry.

jukzi commented:
@akurtakov the build fails. can you please help:
`11:51:16.448 [ERROR] Failed to execute goal on project org.eclipse.e4.ui.tests.css.core: Could not resolve dependencies for project org.eclipse.platform:org.eclipse.e4.ui.tests.css.core:eclipse-test-plugin:1.301.400-SNAPSHOT: Failed to collect dependencies at org.junit.platform:junit-platform-runner:jar:1.9.1 -> junit:junit:jar:4.13.2 -> org.hamcrest:hamcrest-core:jar:1.3: Failed to read artifact descriptor for org.hamcrest:hamcrest-core:jar:1.3: Could not transfer artifact org.hamcrest:hamcrest-core:pom:1.3 from/to eclipse-hosted (https://repo.eclipse.org/content/repositories/eclipse/): authorization failed for https://repo.eclipse.org/content/repositories/eclipse/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.pom, status: 403 Forbidden -> [Help 1]`

jukzi commented:
the added Test does not show up in https://ci.eclipse.org/platform/job/eclipse.platform.ui/job/PR-84/6/testReport/org.eclipse.ui.tests.markers/
Therefore i will add it to InternalTestSuite

jukzi committed:
add test 2 suite

jukzi committed:
Version bump org.eclipse.ui.ide

Touched MarkerSupportView so that the API analysis understands API has
changed.
